README.md

```diff
@@ -522,6 +522,26 @@ Notable exceptions:
   given to the umask.
 + There is no "quiet" option since default behavior is to run silent.
 
+### touch([options ,] file)
+Available options:
```

Comment:
Can you make these modifications to `src/test.js` and then run `scripts/generate-docs.js`? Also, please remove the long options (i.e. `--atime`) since these aren't supported.